Formula/ponyc.rb

```diff
@@ -17,6 +17,15 @@ class Ponyc < Formula
   depends_on "pcre2"
   needs :cxx11
 
+  # https://github.com/ponylang/ponyc/issues/1274
+  # https://github.com/Homebrew/homebrew-core/issues/5346
+  pour_bottle? do
+    reason <<-EOS.undent
+    The bottle requires Xcode 8 to work properly.
```

Comment:
Indent by two-characters. Also, technically isn't this checking for either the Xcode or CLT clang version so the messaging should be adjusted accordingly.